lisa/operating_system.py

```diff
@@ -1460,32 +1460,22 @@ def _is_package_in_repo(self, package: str) -> bool:
         return 0 == result.exit_code
 
     def _get_information(self) -> OsInformation:
-        # The higher version above 7.0 support os-version.
-        try:
-            information = super(Fedora, self)._get_information()
-
-            # remove Linux Server in vendor
-            information.vendor = get_matched_str(
-                information.vendor, self.__vendor_pattern
-            )
-        except Exception:
-            # Parse /etc/redhat-release to support 6.x and 8.x. Refer to
-            # examples of __legacy_redhat_information_pattern.
-            cmd_result = self._node.execute(
-                cmd="cat /etc/redhat-release", no_error_log=True, expected_exit_code=0
-            )
-            full_version = cmd_result.stdout
-            matches = self.__legacy_redhat_information_pattern.match(full_version)
-            assert matches, f"cannot match version information from: {full_version}"
-            assert matches.group("vendor")
-            information = OsInformation(
-                version=self._parse_version(matches.group("version")),
-                vendor=matches.group("vendor"),
-                release=matches.group("version"),
-                codename=matches.group("codename"),
-                full_version=full_version,
-            )
-
+        cmd_result = self._node.execute(
```

Comment:
What's the reason to change it back to redhat-release? The redhat-release may be removed in future. We can use redhat-release as first try, but the os-release should be in except to prevent it fails in future.

Reply:
for some redhat and centos distro, the release got from os-release is 8. but actually it is 8.5, okay, I will change the code accordingly.

Reply:
updated